inference-engine/src/mkldnn_plugin/memory_desc/cpu_memory_desc_utils.h

```diff
@@ -21,6 +21,8 @@ class MKLDNNMemory;
 
 class MemoryDescUtils {
```

Comment:
minor
maybe better `class MemoryDescUtils` -> `namespace MemoryDescUtils` ?

Reply:
Let's leave it as it is, we can always change it later.